src/main/scala/com/github/traviscrawford/spark/dynamodb/BaseScanner.scala

```diff
@@ -1,21 +1,16 @@
 package com.github.traviscrawford.spark.dynamodb
 
-import com.amazonaws.auth.AWSCredentials
-import com.amazonaws.auth.AWSCredentialsProvider
-import com.amazonaws.auth.AWSStaticCredentialsProvider
-import com.amazonaws.auth.BasicAWSCredentials
+import com.amazonaws.auth.{AWSCredentialsProvider, AWSStaticCredentialsProvider, BasicAWSCredentials}
```

Comment:
this is a naive question, but why was the `AWSCredentials` package imported in the first place -- I verified it wasn't used. just a mistake originally, or does that serve some other purpose in scala?

Reply:
A bit confused - where do you see it not being used?

Reply:
@RZachLamberty in Scala,  when you are using a class / object under a package,  you don't need to include the package name before class/object name.